src/components/performance.cjsx

```diff
@@ -14,8 +14,33 @@ Performance = React.createClass
   propTypes:
     courseId: React.PropTypes.string.isRequired
 
+  getInitialState: ->
+    sortOrder: 'is-ascending'
+    sortIndex: 0
+    isNameSort: true
+
+  sortClick: (event) ->
+    isActiveSort =
+      _.contains(event.target.classList, 'is-ascending') or
```

Comment:
Default sorting by student last names is nearly universal in LMS, so we should follow that pattern.

-------
Jason Holmes, Ph.D.
User Experience Manager
OpenStax | OpenStax.org
(713) 893-1326


> On Jun 11, 2015, at 5:19 PM, Philip Schatz <notifications@github.com> wrote:
> 
> In src/components/performance.cjsx <https://github.com/openstax/tutor-js/pull/386#discussion_r32274673>:
> 
> > @@ -14,8 +14,33 @@ Performance = React.createClass
> >    propTypes:
> >      courseId: React.PropTypes.string.isRequired
> >  
> > +  getInitialState: ->
> > +    sortOrder: 'is-ascending'
> > +    sortIndex: 0
> > +    isNameSort: true
> > +
> > +  sortClick: (event) ->
> > +    isActiveSort =
> > +      _.contains(event.target.classList, 'is-ascending') or
> I believe you can just do event.target.classList.contains(...)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/openstax/tutor-js/pull/386/files#r32274673>.
>